docs/hazmat/primitives/asymmetric/dh.rst

```diff
@@ -143,6 +143,9 @@ Key interfaces
 
     .. versionadded:: 0.9
 
+    DH private key objects have methods from this interface as well as
+    :class:`DHPrivateKeyWithSerialization`.
```

Comment:
What about `DH private keys that are not opaque keys (link) also implement DHPrivateKeyWithSerialization to provide serialization methods`? Then opaque keys will link to a glossary section that would say:

An opaque key is a type of key that allows you to perform cryptographic operations such as encryption, decryption, signing, and verification, but does not allow access to the key itself. Typically an opaque key is loaded from a hardware security module (HSM) (link to wikipedia).